LINK/etc/profile.d/evm.sh

```diff
@@ -2,3 +2,4 @@
 
 # Aliases:
 alias vmdb='cd /var/www/miq/vmdb'
+alias appliance='[[ -n ${APPLIANCE_REPO_DIRECTORY} ]] && cd ${APPLIANCE_REPO_DIRECTORY}'
```

Comment:
Also, I added an appliance alias because I can't remember where things are and it's much easier to `vmdb` and `appliance` to switch back and forth between the directories.